userguide/tutorials/metrics-datadog.adoc

```diff
@@ -3,60 +3,28 @@
+== Overview
 
-Measurements and metrics are an important part of managing your Kill
-Bill system to be sure that performance is optimized and any problems
-can be detected early. Kill Bill collects over 1,300 metrics to show you
-every conceivable view of your system's behavior.
-
-Several third party tools are available to manage and display metrics of
-this type. One popular tool is https://www.datadoghq.com/[Datadog]. This
+Several third party tools are available to manage and display Kill Bill's metrics. One popular tool is https://www.datadoghq.com/[Datadog]. This
 package offers a low-cost option to monitor and display whatever metrics
 you may find useful from the Kill Bill collection, including metrics
 that show the performance and resource use of Kill Bill and Kaui
 themselves, and metrics describing the infrastructure on which these
 systems run.
 
-Datadog can be integrated with several types of Kill Bill installations. In this guide we discuss the use of datadog to manage metrics produced
-by two types of implementations on Amazon Web Services (AWS):
-Single Tier and Multitier. With the latest Amazon Machine Image (AMI)
-for Kill Bill, it is very easy to direct the Kill Bill metrics to
-Datadog and work with them in the way that best suits your needs.
-
-== Check Your Kill Bill Version
-
-We assume that you have a working Kill Bill implementation running on
-AWS, using either the
-https://docs.killbill.io/latest/aws-singletier.html[Single Tier] or
-https://docs.killbill.io/latest/aws-multitier.html[Multitier]
-installation options. The single tier option runs Kill Bill, Kaui, the
-MariaDB database and the nginx load balancer on a single AWS EC2
-instance. This is suitable for quick deployment and for testing and
-development. The multitier option runs Kill Bill and Kaui on two or more
-separate EC2 instances, and uses an AWS RDS database manager along with
-an AWS load balancer. This is an attractive option for many production
-uses.
-
-You should be running Kill Bill version 2021.9 or later. This version
-can be configured to provide its full collection of Kill Bill metrics to
-Datadog (and several other analysis tools). If you open the Kill Bill
-admin page in your browser, it should show that you are
-running the "Kill Bill Premium edition":
-
-image::metrics-kb-admin.png[align=center]
+Datadog can be integrated with several types of Kill Bill installations. In this document we discuss the use of Datadog to manage metrics produced
+by single-tier and multi-tier implementations on Amazon Web Services (AWS).
+With the latest Amazon Machine Image (AMI)
+for Kill Bill, the application is preconfigured to collect these metrics and to direct them to Datadog once authorization is provided.
 
+To set up Datadog to manage the metrics available from Kill Bill, there
+are just two steps: (1) setup a Datadog account, and (2)
+edit a configuration file to enable the transfer of the complete Kill Bill metrics.
 
```

Comment:
Maybe split the steps into two separate lines to improve readability

Reply:
Done.